.github/workflows/generate-site.yaml

```
@@ -46,7 +46,7 @@ jobs:
       success: ${{ steps.build-site.outputs.success }}
       is-bot: ${{ steps.is-bot.outputs.bot }}
     container:
-      image: ghcr.io/hubverse-org/hub-dash-site-builder:latest
+      image: ghcr.io/hubverse-org/hub-dash-site-builder:main
```

change back to latest because https://github.com/hubverse-org/hub-dash-site-builder/pkgs/container/hub-dash-site-builder/386287611?tag=v1.0.0 has been published

```suggestion
      image: ghcr.io/hubverse-org/hub-dash-site-builder:latest
```